addons/stock/models/stock_move.py

```diff
@@ -2113,6 +2113,8 @@ def _trigger_scheduler(self):
                 ('location_id', 'parent_of', move.location_id.id),
                 ('company_id', '=', move.company_id.id),
                 '!', ('location_id', 'parent_of', move.location_dest_id.id),
+                '|', ('snoozed_until', '=', False),
+                ('snoozed_until', '<=', fields.Date.today()),
```

Comment:
@Whenrow could you change fields.Date.today() by fields.Date.context_today(self)